### ISSUE TITLE
Brotli Speed for responses (Nest or any other server)

### ISSUE THREAD

**Issue by orenmizr** [fastify/help#889]
from nest docs: "While Brotli is quite efficient in terms of compression ratio, it's also quite slow. Due to this, you may want to tell fastify-compress to only use deflate and gzip to compress responses; you'll end up with larger responses but they'll be delivered much more quickly."

reported 10x slower than gzip. 

is that true  ? i did not expect to not use brotli.

**Comment by Uzlopak** [comment]
Yes this is true. Brotli compresses more aggressive than gzip. You should consider brotli for static files + precompressing those static files. For dynamic ascii/non-binary data, like rest api calls, you should consider gzjp, as it compresses faster and for small non-binary files it has imho no significant size difference to brotli.

Its the same with zopfli, which is a more aggressive gzip implementation. So using zopfli is maybe good for precompressing static files, but for dynamic data you would bottleneck your app for saving few bytes.
